#!/bin/sh /usr/share/dpatch/dpatch-run
## 325-magic-add-ssl.dpatch by Nicolas Collignon <tsointsoin@gmail.com>
##
## DP: New magic for OpenSSH key files (Closes: #439537).

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/ssh file-4.21/magic/Magdir/ssh
--- file-4.21.orig/magic/Magdir/ssh	1970-01-01 00:00:00.000000000 +0000
+++ file-4.21/magic/Magdir/ssh	2007-08-26 08:25:03.000000000 +0000
@@ -0,0 +1,8 @@
+# Type: OpenSSH key files
+# From: Nicolas Collignon <tsointsoin@gmail.com>
+
+0	string	SSH\ PRIVATE\ KEY	OpenSSH RSA1 private key,
+>28	string	>\0			version %s
+
+0	string	ssh-dss\		OpenSSH DSA public key
+0	string	ssh-rsa\		OpenSSH RSA public key
